doc/getting_started_LAN9662.rst

```diff
@@ -304,7 +304,7 @@ log::
     ANA_RT_VLAN_PCP[1].PCP_MASK                                                                                       = 0x00000000 -> 0x000000ff
     ANA_RT_VLAN_PCP[1].VLAN_ID                                                                                        = 0x00000000 -> 0x00000000
     ANA_RT_VLAN_PCP[1].VLAN_PCP_ENA                                                                                   = 0x00000000 -> 0x00000001
-    vcap add is1 10 3 s1_rt first 0 rt_vlan_idx 1 0x7 l2_mac 12:A9:2D:16:93:83 ff:ff:ff:ff:ff:ff rt_type 1 0x3 rt_frmid 32769 0xffff s1_rt rtp_id 5 fwd_ena 1 fwd_mas0
+    vcap add 10000 is1 10 0 VCAP_KFS_RT RT_VLAN_IDX 1 0x7 L2_MAC 12:A9:2D:16:93:83 ff:ff:ff:ff:ff:ff RT_TYPE 1 0x3 RT_FRMID 32769 0xffff VCAP_AFS_S1_RT RTP_ID 5 FWD_ENA 1 FWD_MASK 0
```

Comment:
This does not match with the updated script. FWD_MASK is 0x10 in the script.
Additionally the vcam_handle is not what I expect.